nsxt/resource_nsxt_vm_tags_test.go

```diff
@@ -70,6 +73,68 @@ func TestAccResourceNsxtVMTags_import_basic(t *testing.T) {
 	})
 }
 
+func TestAccResourceNsxtVMTags_withSpecificLogicalPort(t *testing.T) {
+	vmID := getTestVMID()
+	logicalSwitchID := getTestLogicalSwitchID()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccEnvDefined(t, "NSXT_TEST_VM_ID")
+			testAccEnvDefined(t, "NSXT_TEST_LOGICAL_SWITCH_ID")
```

Comment:
Can this please be renamed to NSXT_TEST_VM_LOGICAL_SWITCH_ID for clarity? Same below